Sources/Classes/Public/RSDeviceInfo.h

```diff
@@ -12,6 +12,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface RSDeviceInfo : NSObject
 
+- (NSString *) getDeviceType;
```

Comment:
If this method only required for this class. Don't need to expose as public. And make it static method.

Reply:
Okay, got it. 
Also removed this method in the latest commit as we are setting context.device.type programatically